src/averageAges.js

```diff
@@ -15,11 +16,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
+    ? people.filter((p) => p.sex === 'm' && Math.ceil(p.died / 100) === century)
```

Comment:
[Destructuring](https://mate-academy.github.io/style-guides/javascript.html#destructuring) can be used.
```suggestion
    ? people.filter(({ sex,  died }) => sex === 'm' && Math.ceil(died / 100) === century)
```